pytorch_lightning/logging/comet_logger.py

```diff
@@ -1,25 +1,94 @@
 try:
     from comet_ml import Experiment as CometExperiment
+    from comet_ml.papi import API
 except ImportError:
     raise ImportError('Missing comet_ml package.')
 
+from torch import is_tensor
+
 from .base import LightningLoggerBase, rank_zero_only
 
 
 class CometLogger(LightningLoggerBase):
-    def __init__(self, *args, **kwargs):
-        super(CometLogger, self).__init__()
-        self.experiment = CometExperiment(*args, **kwargs)
+    def __init__(self, api_key, workspace, rest_api_key=None, project_name=None, experiment_name=None, **kwargs):
+        """
+        Initialize a Comet.ml logger
+
+        :param api_key: API key, found on Comet.ml
+        :param workspace: Name of workspace for this user
+        :param project_name: Optional. Send your experiment to a specific project.
+        Otherwise will be sent to Uncategorized Experiments.
+        If project name does not already exists Comet.ml will create a new project.
+        :param rest_api_key: Optional. Rest API key found in Comet.ml settings. This is used to determine version number
+        :param experiment_name: Optional. String representing the name for this particular experiment on Comet.ml
+        """
+        super().__init__()
+        self._experiment = None
+
+        self.api_key = api_key
+        self.workspace = workspace
+        self.project_name = project_name
+
+        self._kwargs = kwargs
+
+        if rest_api_key is not None:
```

Comment:
you may just write:
`if rest_api_key:`

Reply:
I disagree on this one. The current version is more explicit.